app/globals.css

```
@@ -320,6 +320,20 @@ button.word-card:focus-visible {
   letter-spacing: 0.09em;
 }
 
+.word-card-score {
+  position: absolute;
+  right: 0.7rem;
+  bottom: 0.65rem;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)